go.mod

```diff
@@ -25,8 +25,8 @@ require (
 	gomodules.xyz/jsonpatch/v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
 	helm.sh/helm/v3 v3.16.3
-	istio.io/client-go v1.24.0-alpha.0.0.20250103213757-fb95213c2bc2
-	istio.io/istio v0.0.0-20250119170654-0c5460eeba8a
+	istio.io/client-go v1.24.3-0.20250110022005-e8de2e86eed8
```

Comment:
The latest version in `versions.yaml` is `v1.24.2` should we use that one instead?

Reply:
good point, will do

Reply:
interestingly, that also removed some of the strange generated CRD changes. Good catch!